packages/manager/modules/vps/src/veeam/order/translations/Messages_fr_FR.json

```diff
@@ -3,6 +3,8 @@
   "vps_configuration_veeam_order_back_btn_title": "Retour à la liste des backups automatisés",
   "vps_configuration_veeam_order_step1_loading_error": "Une erreur est survenue lors du chargement des informations",
   "vps_configuration_veeam_order_step1_info": "Pour le premier paiement, vous serez facturé au prorata du nombre de jours restant avant la date de renouvellement de votre VPS.",
+  "vps_configuration_veeam_order_backup_info": "La fonction auto-backup vous permet de restaurer votre VPS à tout moment. Vous disposez, sur votre tableau de bord, de <strong>7 sauvegardes de votre VPS</strong> (hors disques additionnels) <strong>correspondant aux 7 derniers jours d’utilisation.</strong>",
```

Comment:
i think we should avoid the hardcoded value (`7`).
There is any raison to hardcode them ?

Reply:
It is hard coded since it is not a dynamic value. It's a constant value.

Reply:
If you are sure these values will never change good for me.
But keep in mind if one day we plan to change these values, we will have to update all translations files.
This why i think it's better to have them into `const` file and passed them as parameters.